src/FileObject.h

```diff
@@ -65,6 +67,7 @@ class FileObject {
 	virtual int				Sort();
 
 	static int				SortVector(FOVector & foVect);
+	WORD					m_cutpaste;

```

Comment:
Is this variable used?

Reply:
Yes, if an object is part of a drag operation it holds the information if it is a cut or copy operation